component/admin/views/icalevent/tmpl/overview.php

```diff
@@ -170,9 +170,14 @@
 							}
 							else
 							{
+
+								$firstRepeat = $row->getFirstRepeat();
 								$times = '<table style="border: 1px solid #666666; width:100%;">';
-								$times .= '<tr><td>' . JText::_('JEV_FROM') . ' : ' . ($row->alldayevent() ? JString::substr($row->publish_up(), 0, 10) : JString::substr($row->publish_up(),0,16)) . '</td></tr>';
+								$times .= '<tr><td>' . JText::_('JEV_FROM') . ' : ' . ($firstRepeat->alldayevent() ? JString::substr($firstRepeat->publish_up(), 0, 10) : JString::substr($firstRepeat->publish_up(),0,16)) . '</td></tr>';
 								$times .= '<tr><td>' . JText::_('JEV_TO') . ' : ' . (($row->noendtime() || $row->alldayevent()) ? JString::substr($row->publish_down(), 0, 10) : JString::substr($row->publish_down(),0,16)) . '</td></tr>';
+								if ($row->hasrepetition()) {
+									$times .= '<tr><td>' . JText::_('JEV_NEXT_REPEAT') . ' : ' . ($row->alldayevent() ? JString::substr($row->publish_up(), 0, 10) : JString::substr($row->publish_up(),0,16)) . '</td></tr>';
```

Comment:
No - I think its important not to add an extra row of info when it adds nothing.  I think we should change your line 178 to read 

`if ($row->hasrepetition() && $firstRepeat->publish_up() != $row->publish_up()) {`

That way it gives the extra information when needed.

Reply:
I actually think this will add in confusion, but as you wish.